frontend/src/app/rb-ngb/modal.service.ts

```diff
@@ -24,9 +24,9 @@ export class Modal<T> {
   }
 }
 
-// not provided in root, otherwise it doesn't work properly when used in a lazy module.
-// See https://github.com/ng-bootstrap/ng-bootstrap/issues/3784
-@Injectable()
+@Injectable({
```

Comment:
So I guess the issue above is now fixed?

Reply:
I did some extensive testing on the app, and I think there is no issue.
The only form I found in a modal is the CSV import on an order, and it works perfectly 👍